IntelFsp2Pkg/Include/FspEas/FspApi.h

```diff
@@ -57,12 +141,12 @@ typedef struct {
   /// Pointer to the non-volatile storage (NVS) data buffer.
   /// If it is NULL it indicates the NVS data is not available.
   ///
-  UINT32                      NvsBufferPtr;
+  VOID                        *NvsBufferPtr;

```

Comment:
This might break the current x64 boot.

Reply:
Do you want to change all of the VOID* to UINT32? Or if there is a way to use an override package for x64 build? Or some other approach?

Reply:
At this point, let us change all  VOID* and function pointer such as FspEventHandler into UINT32 so as to allow x64 SBL to be able to call 32bit FSP binary properly.

Reply:
I think I got them all, all of the pointers inside of the structures but not in the function pointer typedef's, is that right?